Library/Homebrew/software_spec.rb

```diff
@@ -441,11 +441,29 @@ def checksums
     end
     tags.reverse.map do |tag|
       {
-        collector[tag][:checksum] => tag,
-        cellar: collector[tag][:cellar],
+        "tag"    => tag,
+        "digest" => collector[tag][:checksum],
+        "cellar" => collector[tag][:cellar],
       }
     end
   end
+
+  def generate_sha256_line(tag, digest, cellar)
+    sha256_line = "sha256 \"#{digest}\" => :#{tag}"
+    return sha256_line if cellar.blank?
+
+    default_cellars = [
+      Homebrew::DEFAULT_MACOS_CELLAR,
+      Homebrew::DEFAULT_MACOS_ARM_CELLAR,
+      Homebrew::DEFAULT_LINUX_CELLAR,
+    ]
+    if cellar.is_a? Symbol
+      sha256_line += ", :cellar => :#{cellar}"
+    elsif default_cellars.exclude?(cellar)
+      sha256_line += ", :cellar => \"#{cellar}\""
+    end
+    sha256_line
+  end
```

Comment:
Here is my proposal for generating `sha256` lines in `dev-cmd/bottle.rb` instead of `software_spec.rb`: https://github.com/SeekingMeaning/brew/commit/50efc59f4

Reply:
Cool, I though about it before, and I went for the easy solution, because I was unsure how to modify `bottle_output` to make that work. Thanks for the help, I made that change too.